transformation/v2-to-fhirr4-service/README.md

```diff
@@ -29,6 +29,34 @@ You do not have to write code from scratch but reuse these existing services whe
     --data-raw 'MSH|^~\&|EPIC|EPICADT|SMS|SMSADT|202211031408|CHARRIS|ADT^A01|1817457|D|2.8'
     ```
 
+## Customization
+
+You can customize the v2 segments to FHIR transformation logic by implementing a service with custom mappings and configuring the v2-to-fhirr4-service to invoke the custom service. This custom service can be implemented in [Ballerina](https://ballerina.io/) by using the provided template, or you can use the OAS definition (resources/custom-mapping-service-openapi.yaml) to implement your service using any other technology. The following is the command to initiate the Ballerina template:
+
+```bash
+bal new <service_name> -t ballerinax/health.fhir.templates.r4.v2tofhircustomsvc
+```
+
+You can configure the v2-to-fhir-service to point to your custom service to invoke the mapping endpoints. The following is a sample configuration structure to invoke a custom service.
+
+```toml
+[serviceConfig]
+baseUrl = "http://localhost:9091/v2tofhir"
+
+[serviceConfig.segmentToAPI]
+NK1 = "/segment/nk1"
+PID = "/segment/pid"
+```
+
+
+| Table | Field | Description |
+|-------|-------|-------------|
+| `serviceConfig`| `baseUrl` | The base URL of the custom service. |
+| `serviceConfig.segmentToAPI`| `<segmentName>` | The path of the custom service endpoint against the each customized segment name. |
+
+> **Note:** If you deploy this to [Choreo](https://wso2.com/choreo/), you can apply configurations through the [config editor](https://wso2.com/choreo/docs/devops-and-ci-cd/manage-configurations-and-secrets/#manage-ballerina-configurables) in the Choreo console.
```

Comment:
If you deploy this to [Choreo](https://wso2.com/choreo/) (refer to the ## [Optional] Deploy in Choreo section), ...